xoshiro256plusplus/test/same-bits-ll/dune

```diff
@@ -1,4 +1,4 @@
-(executable
- (name sameBitsLL)
- (libraries
-  xoshiro.256++.pure xoshiro.256++.bindings))
```

Comment:
@Niols I think the issue you faced was that this was declaring an executable and not a test

Reply:
Yes, I think you are right. I must have forgotten that one of them was an executable (it was because it wasn't yet really in the shape of a test, but oh well.